### DISCUSSION TITLE
Missing Cities and Regions for country_code TZ

### DISCUSSION THREAD

**Discussion by doublezze** [dr5hn/countries-states-cities-database#305] [General]
Hi Darshan and Team, the database is fantastic! great work!

Browsed database and noticed two Regions (Mbeya and Songwe) are missing  in states and cities.

I don't know if this a data issue or the reason for the omission.
Thanks
doublezze

**Comment by dr5hn** [comment]
@doublezze Thanks for reporting. Will take a look at it soon.

**Reply by doublezze** [reply]
Thanks for the prompt response. I'll look forward to your findings

**Reply by dr5hn** [reply]
@doublezze Check #310 

I have added (Mbeya and Songwe) regions. However, I was able to find the cities for Mbeya and have added them.

But, Did not able to find the cities of Songwe. Can you please help me find the cities of Songwe?

**Reply by doublezze** [reply]
Hi Darshan,
Mbeya was split into 2 regions Mbeya and Songwe in 2016 or there about.
[mbysongweregions.txt](https://github.com/dr5hn/countries-states-cities-database/files/7325426/mbysongweregions.txt)

Both region's towns are attached.

I also couldn't find Bermuda within North America.

Thanks
